CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 ### Features
 - [#304](https://github.com/influxdata/influxdb-client-go/pull/304) Added public constructor for `QueryTableResult`
 - [#307](https://github.com/influxdata/influxdb-client-go/pull/307) Synced generated server API with the latest [oss.yml](https://github.com/influxdata/openapi/blob/master/contracts/oss.yml). 
+- [#308](https://github.com/influxdata/influxdb-client-go/pull/308) Added Flux query parameters. Supported by InfluxDB Cloud only now.
+- [#308](https://github.com/influxdata/influxdb-client-go/pull/308) Go 1.17 is required
```

Comment:
Does it depend on 1.17 features? Isn't it the case that 1.17 is now used by tests, so it is not a feature.

Reply:
It should IMHO at least follow [Go Security Policy](https://go.dev/security) so that each major Go release is supported until there are two newer major releases.

Reply:
Thank you for the explanation. 1.18 is likely to come this month and 1.16 will thus go out of support.  So it makes sense to require 1.17 herein.